File: python/1060_numeros_positivos.py
a = float(input())
b = float(input())
c = float(input())
d = float(input())
e = float(input())
f = float(input())

N = [a, b, c, d, e, f]

contador = 0

for num in N:
    if num > 0:
        contador += 1

print(f'{contador} valores positivos')

File: c/c99/1060_numeros_positivos.c
#include <stdio.h>

int main() {
    int positivos = 0;
    float N;
    for (int i = 1; i <= 6; i++) {
        scanf("%f", &N);
        if (N > 0) {
            positivos++;
        }
    }
    printf("%d valores positivos\n", positivos);
    return 0;
}
